### PR TITLE
Sm/schema nuts

### PR DESCRIPTION
**Pull request by mshanemc**
### What does this PR do?
add NUTS [release streamlining, just-nuts, and because they're good]
adds NUTs to the CI

### What issues does this PR fix or reference?
@W-9299637@

### CODE REVIEW

File: test/schema.nut.ts [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    'triggerable',
+    'undeletable',
+    'updateable',
+  ];
+
+  before(async () => {
+    // validate that we can auth to a hub from env
+    if (!process.env.TESTKIT_ORG_USERNAME && !process.env.TESTKIT_HUB_USERNAME && !process.env.TESTKIT_AUTH_URL) {
+      throw new Error('Set environment variables so testkit can auth to a hub');
+    }
```
Comment (WillieRuemmele) [inline test/schema.nut.ts:42]: this should also be in TestKit right?

**Review by WillieRuemmele — APPROVED** [review]

File: test/schema.nut.ts (hunk 1 above)
Comment (mshanemc) [reply test/schema.nut.ts:42]: there was some discussion about that in slack yesterday.  I'm not sure Testkit will know about the NUTs requirements.  For example, plugin-config NUTs need no hub.  Plugin-auth NUTs need every possible auth method.

Comment (mshanemc) [reply test/schema.nut.ts:42]: the nicest thing testkit would do would be to have some boilerplate options (requiresHub or requiresWhatever) that NUT authors could reuse, I guess?

Comment (shetzel) [reply test/schema.nut.ts:42]: Long ago I was thinking of adding a `ensureEnv` option to TestSession that would be an array of required env vars but decided it was just as easy for NUTs to do that themselves if they wanted.

**Review by shetzel — APPROVED** [review]

**Pull request merged**